app/models/hospital.rb

```diff
@@ -1,3 +1,11 @@
 class Hospital < ApplicationRecord
   has_many :doctors
+  has_many :patients, through: :doctors
+   
+  validates_presence_of :name
+  
+  def doctors_ordered_by_patients
+    # didnt get to finish this one :(
+  end
```

Comment:
I understand wanting to get to the extension, but when using a TDD programming style, your tests should have been written first. :)